libs/StateSync.Managed/src/InternedBlobRef.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.MixedReality.Sharing.StateSync
+{
+    /// <summary>
+    /// A lightweight unallocated reference to an interned blob.
+    /// Can be converted to an allocated <see cref="InternedBlob"/>, <see cref="AsInternedBlob"/>.
```

Comment:
nit: not sure that "unallocated" and "allocated" are the right terms, I'd use "stack-allocated" and "heap-allocated".

Reply:
These comments are being copy-pasted around since the Andrei's draft. I guess it's time to fix them.